### ISSUE TITLE
How to increase the timeout of the workspace?

### ISSUE THREAD

**Issue by josefredohd** [eclipse-che/che#16621]
I am using eclipse che in google chrome, but it stops due to inactivity

**Comment by sleshchenko** [comment]
@josefredohd the way how you can increase it depends on the way you deployed Che:
**Helm**
```
kubectl patch configmap che --patch "{\"spec\":{\"data\":{\"CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT\": \"-1\"}}}" --type=merge -n che
kubectl rollout restart deployment/che -n che
```
**Operator:**
```
kubectl patch checluster/eclipse-che --patch "{\"spec\":{\"server\":{\"customCheProperties\": {\"CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT\": \"-1\"}}}}" --type=merge -n che
```

P.S. There is -1 - not limited, but feel free to assign any value in milliseconds (the default value 1800000 (30minutes)).

It must be documented!

**Comment by josefredohd** [comment]
@sleshchenko How can I do it from the chrome browser?

**Comment by sleshchenko** [comment]
@josefredohd you're able to just execute commands I shared in the terminal. I'm not able to provide you instructions on how you can update it from chrome, since I don't know which platform you use, k8s/OS, which version...

**Comment by amisevsk** [comment]
Also note that if this is referring to che.openshift.io, the inactivity timeout is set at 30 minutes and cannot be modified as this is a Che server setting.

**Comment by josefredohd** [comment]
@sleshchenko I am using the k8s platform

**Comment by codingkriggs** [comment]
I am using gcloud + k8s + helm. So for the helm scenario described by @sleshchenko , I had to remove the outer "spec" part of the json, because if I followed those instructions, the first command would return:
```
configmap/che patched (no change)
```
So, to rephrase it to what I did to get it to update properly:
```
kubectl patch configmap che --patch "{\"data\":{\"CHE_LIMITS_WORKSPACE_IDLE_TIME
OUT\": \"-1\"}}" --type=merge -n che
kubectl rollout restart deployment/che -n che
```

**Comment by sleshchenko** [comment]
@codingkriggs thanks for providing the fix. I've updated my comment as well to avoid confusion.

**Comment by tomgeorge** [comment]
Also, documentation for editing the `CheCluster` CR in the case of an operator installation, and the `che` `ConfigMap` in the case of a helm installation are at [advanced configuration options for the che server component](https://www.eclipse.org/che/docs/che-7/installation-guide/advanced-configuration-options-for-the-che-server-component/).  It is under Installation Guide -> Configuring Che -> Advanced Configuration Options for Che Server.  Maybe this page belongs in the Administration Guide, or not nested so deeply?

**Comment by boczkowska** [comment]
Docs Jira issue https://issues.redhat.com/browse/RHDEVDOCS-2581
